src/transport/proto.rs

```diff
@@ -132,3 +143,54 @@ pub enum EdgeStatus {
     Up,
     Down,
 }
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct AlertMessage {
```

Comment:
this already exists in the file https://github.com/LucioFranco/clique/blob/master/src/transport/proto.rs#L121

Reply:
Huh, didn't realize that. Maybe we should use `AlertMessage` to keep naming parity with the protocol messages? It would certainly reduce the confusion.